Makefile

```diff
@@ -17,11 +17,19 @@ test:
 
 # Incremental test
 itest:
-	@env DOMAIN=$(DOMAIN) SAUCE_USERNAME=$(SAUCE_USERNAME) SAUCE_ACCESS_KEY=$(SAUCE_ACCESS_KEY) $(MOCHA) test/
+	@env DOMAIN=$(DOMAIN) SAUCE_USERNAME=$(SAUCE_USERNAME) SAUCE_ACCESS_KEY=$(SAUCE_ACCESS_KEY) $(MOCHA) --no-timeouts test/
 
 
 # removes all SauceLabs jobs and linked assets
 # removes all local screenshots
 clean:
-	@env SAUCE_USERNAME=$(SAUCE_USERNAME) SAUCE_ACCESS_KEY=$(SAUCE_ACCESS_KEY) node scripts/clean.js
-	@rm -rf screenshot/*.png
+	@env SAUCE_USERNAME=$(SAUCE_USERNAME) SAUCE_ACCESS_KEY=$(SAUCE_ACCESS_KEY) node scripts/delete_job.js
+	@rm -rf screenshot/
+
+
+# Download all assets
+asset:
+	@mkdir -p screenshot
+	@env SAUCE_USERNAME=$(SAUCE_USERNAME) SAUCE_ACCESS_KEY=$(SAUCE_ACCESS_KEY) node scripts/get_asset.js
+	@chmod +x asset.sh && ./asset.sh
```

Comment:
It can be either, I opted not to use the symbolic representations as the user will create the asset.sh file (./scripts/get_assets.js) and would already own / have permissions to it.